<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>ComboBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>普通ComboBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btn_initCboBox">
               <property name="text">
                <string>初始化列表</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_clearCboBox">
               <property name="text">
                <string>清除列表</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_isEditable">
               <property name="text">
                <string>可编辑</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="cbb_generaCboBox">
             <item>
              <property name="text">
               <string>北京</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>上海</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>关联用户数据的ComboBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_initCityNumber">
             <property name="text">
              <string>初始化城市+区号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbb_userDataCboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>QPlainTextEdit(具有标准快捷菜单)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_addContextToCboBox">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>文本框内容添加到ComboBox</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clearContext">
            <property name="text">
             <string>清除文本内容</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_isReadOnly">
            <property name="text">
             <string>Read Only</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="pte_plainTextEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
